### PR TITLE
refactor: update stack with component-rollup-config

### PR DESCRIPTION
**Pull request by cbourgois**
### Description of the Change

* Remove documentation & tests
* replace grunt by rollup (using `@ovh-ux/component-rollup-config`)
* Fix dependencies

BREAKING CHANGE: module name is now `ngTailLogs` and package is scoped in `@ovh-ux`

    Before:

        import 'ovh-angular-tail-logs';
        angular.module('myModule', ['ovh-angular-tail-logs'])

    After:

        import '@ovh-ux/ng-tail-logs';
        angular.module('myModule', ['ngTailLogs'])

19dc591 — refactor: update stack with component-rollup-config

ref:

/cc @jleveugle @frenautvh @antleblanc

### CODE REVIEW

**Review by FredericEspiau — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -1,23 +1,31 @@
-# ovh-angular-tail-logs
+# ng-tail-logs
 
-![OVH component](https://user-images.githubusercontent.com/3379410/27423240-3f944bc4-5731-11e7-87bb-3ff603aff8a7.png)
+> OVH ANgular Tail Logs Module
```
Comment (antleblanc) [inline README.md]:
```suggestion
> OVH Angular Tail Logs Module
```

File: README.md [hunk 2]
```diff
@@ -1,23 +1,31 @@
...
 
-[![NPM](https://nodei.co/npm/ovh-angular-tail-logs.png?downloads=true&downloadRank=true&stars=true)](https://nodei.co/npm/ovh-angular-tail-logs/)
+[![Downloads](https://badgen.net/npm/dt/@ovh-ux/ng-tail-logs)](https://npmjs.com/package/@ovh-ux/ng-tail-logs) [![Dependencies](https://badgen.net/david/dep/ovh-ux/@ovh-ux/ng-tail-logs)](https://npmjs.com/package/@ovh-ux/ng-tail-logs?activeTab=dependencies) [![Dev Dependencies](https://badgen.net/david/dev/ovh-ux/@ovh-ux/ng-tail-logs)](https://npmjs.com/package/@ovh-ux/ng-tail-logs?activeTab=dependencies) [![Gitter](https://badgen.net/badge/gitter/ovh-ux/blue?icon=gitter)](https://gitter.im/ovh/ux)
```
Comment (antleblanc) [inline README.md]:
```suggestion
[![Downloads](https://badgen.net/npm/dt/@ovh-ux/ng-tail-logs)](https://npmjs.com/package/@ovh-ux/ng-tail-logs) [![Dependencies](https://badgen.net/david/dep/ovh-ux/ng-tail-logs)](https://npmjs.com/package/@ovh-ux/ng-tail-logs?activeTab=dependencies) [![Dev Dependencies](https://badgen.net/david/dev/ovh-ux/ng-tail-logs)](https://npmjs.com/package/@ovh-ux/ng-tail-logs?activeTab=dependencies) [![Gitter](https://badgen.net/badge/gitter/ovh-ux/blue?icon=gitter)](https://gitter.im/ovh/ux)
```

File: README.md [hunk 3]
```diff
@@ -1,23 +1,31 @@
...
+## Install
 
+```sh
+yarn add @ovh-ux/ng-tail-logs
+```
 
-[![Build Status](https://travis-ci.org/ovh/ovh-angular-tail-logs.svg)](https://travis-ci.org/ovh/ovh-angular-tail-logs)
+## Usage
 
 ```javascript
```
Comment (antleblanc) [inline README.md]: You can use the shorthand:

```txt
```js
```

File: package.json [hunk 4]
```diff
@@ -1,50 +1,56 @@
...
-  "name": "ovh-angular-tail-logs",
+  "name": "@ovh-ux/ng-tail-logs",
   "version": "1.1.1",
   "description": "Poll a log API for displaying messages",
+  "keywords": [
+    "angular",
+    "log",
+    "ovh"
+  ],
+  "repository": "ovh-ux/ovh-angular-tail-logs",
```
Comment (antleblanc) [inline package.json]: The repository have to be renamed as well.

```suggestion
  "repository": "ovh-ux/ng-tail-logs",
```

File: rollup.config.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import rollupConfig from '@ovh-ux/component-rollup-config';
+
+const config = rollupConfig({
+  input: 'src/index.js',
+});
+
+export default [
```
Comment (antleblanc) [inline rollup.config.js:7]: Provide config.es() by default?

And:

```js
if (process.env.BUILD === 'production') {
  outputs.push(config.cjs());
  outputs.push(congif.umd({…})
}
```

**Review by antleblanc — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (cbourgois) [reply README.md]: Done

**Comment by jleveugle** [comment]
Hello @cbourgois,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**